Riot/Modules/Common/UserIndicators/UserIndicatorPresenter.swift

```diff
@@ -23,6 +23,8 @@ import UIKit
 enum UserIndicatorType {
     case loading(label: String, isInteractionBlocking: Bool)
     case success(label: String)
+    case failure(label: String)
+    case appState(label: String, icon: UIImage?)
```

Comment:
I think calling this `appState` might be overly specific given the generic nature of the `UserIndicatorType` Could this simply be `custom(label: String, icon: UIImage?)`?

Reply:
Makes sense.